main-service/src/main/java/ru/practicum/ewm/app/user/model/UserDtoMapper.java

```diff
@@ -17,4 +18,8 @@ public interface UserDtoMapper {
     UserCommonFullDto toFullDto(User user);
 
     UserOutputShortDto toShortDto(User user);
+
+    SubscriptionCountedUserDto toCountedUserDto(UserOutputShortDto dto, Long count);
+
+    SubscriptionCountedUserDto toCountedUserDto1(SubscriptionCountedUser countedUser);
```

Comment:
Для таких случаев есть [перегрузка методов](https://metanit.com/java/tutorial/2.18.php)
```suggestion
    SubscriptionCountedUserDto toCountedUserDto(SubscriptionCountedUser countedUser);
```

Reply:
ошибка от невнимательности

```
SubscriptionCountedUserDto toCountedUserDto(UserOutputShortDto, Long)
```

не нужен. Сначала я задекларировал его, т.к. засомневался почему то, что мапстракт сгенерит метод c User-ом как аргумент.
Реализовал логику. Потом рефакторил на использование сразу SubscriptionCountedUser.
после рефакторинга и на радостях что проверки проходят:) забыл удалить ненужный абстрактный метод